### PR TITLE
Yet more fixes for the play/pause test page.

### PR DESCRIPTION
**Pull request by ehlemur-zz**

### CODE REVIEW

File: src/pause-play/js/main.js [hunk 1]
```diff
@@ -66,18 +79,21 @@ class PeerConnection {
...
   }
 
   onCreateOfferSuccess(desc) {
     this.localConnection.setLocalDescription(desc);
     this.remoteConnection.setRemoteDescription(desc);
 
-    var onCreateAnswerSuccess = this.onCreateAnswerSuccess.bind(this);
-    this.remoteConnection.createAnswer().then(onCreateAnswerSuccess, logError);
+    this.remoteConnection.createAnswer().then(
+      () => {
```
Comment (KaptenJansson) [inline src/pause-play/js/main.js:94]: shouldn't you add desc as an argument? Also format it the same as on line https://github.com/webrtc/test-pages/pull/7/files#diff-2edb0afc11050f78b4acd671ec919f5eR84

**Review by KaptenJansson — CHANGES_REQUESTED** [review]

File: src/pause-play/js/main.js (hunk 1 above)
Comment (ehlemur-zz) [reply src/pause-play/js/main.js:94]: If I do that, it complains:

  94:10  error  'desc' is already declared in the upper scope                                                                  no-shadow
  94:19  error  Statement inside of curly braces should be on next line                                                        brace-style
  94:52  error  Closing curly brace should be on the same line as opening curly brace or on the line after the previous block  brace-style

Comment (ehlemur-zz) [reply src/pause-play/js/main.js:94]: If I do that, it complains:

  94:10  error  'desc' is already declared in the upper scope                                                                  no-shadow
  94:19  error  Statement inside of curly braces should be on next line                                                        brace-style
  94:52  error  Closing curly brace should be on the same line as opening curly brace or on the line after the previous block  brace-style

Comment (KaptenJansson) [reply src/pause-play/js/main.js:94]: Ok I see, this could be that it does not lint properly for es6 as well. Lets leave it as is for now.

**Review by KaptenJansson — APPROVED** [review]

**Pull request merged**